apps/condo/domains/organization/utils/name.utils.spec.js

```diff
@@ -0,0 +1,23 @@
+const { isValidName } = require('@condo/domains/organization/utils/name.utils')
+
+const VALID_NAMES = ['ООО «УК №9»', 'TOP', 'Рога и копыта']
+const INVALID_NAMES = ['01234556789', '%%%', '!$^&']
+
+// The case when the name length is less than 3 is handled by a minLength validator
+describe('isValidName()', () => {
```

Comment:
undefined, null checks will be awesome. and I guess this should also be in TS

Reply:
Could you also check current real client names table.

Reply:
done